nerdlandbot/helpers/TranslationHelper.py

```diff
@@ -1,6 +1,8 @@
 from discord.ext import commands
+from datetime import datetime
```

Comment:
Unnecessary imports

Reply:
I've fixed it now. As for the eightball_url, I think it's because someone grouped them together on a commit and it then moved downwards as I merged it, as Bob said.